viewer/viewer/static_src/crawsqueal.css

```diff
@@ -1,25 +1,33 @@
+.content_main-alt {
+  border-top: 0 !important;
+  width: 100% !important;
+}
+
 .search_results .results_count {
   padding-left: 1.875em;
+  padding-bottom: 15px;
+  margin-top: 30px;
+  height: 70px;
```

Comment:
This height causes a problem if there are no results:

<img width="1178" alt="image" src="https://user-images.githubusercontent.com/654645/180517735-27b3e6b3-3875-44d5-b62a-6017e4cc3c3b.png">